Kite SDK/PSPrintSDK/OLCountry.h

```diff
@@ -18,4 +18,6 @@
 + (OLCountry *)countryForCode:(NSString *)code;
 + (OLCountry *)countryForCurrentLocale;
 
+- (BOOL)isEUCountry;
```

Comment:
Nope, go ahead, you are right.

> On Jun 3, 2015, at 4:57 PM, Deon Botha <notifications@github.com> wrote:
> 
> In Kite SDK/PSPrintSDK/OLCountry.h <https://github.com/OceanLabs/iOS-Print-SDK/pull/138#discussion_r31638580>:
> 
> > @@ -18,4 +18,6 @@
> >  + (OLCountry *)countryForCode:(NSString *)code;
> >  + (OLCountry *)countryForCurrentLocale;
> >  
> > +- (BOOL)isEUCountry;
> Would you be adverse to me changing this to:
> 
> @property <https://github.com/property> (nonatomic, readonly, getter=isInEurope) BOOL inEurope;
> 
> mainly because EU means something specific and Europe continent is more general
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/OceanLabs/iOS-Print-SDK/pull/138/files#r31638580>.
>